packages/web3torrent/src/components/channel-list/ChannelList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {Channel} from './channel/Channel';
+import {ChannelState} from '../../clients/web3t-channel-client';
+import './ChannelList.scss';
+
+export type ChannelListProps = {channels: Array<Partial<ChannelState>>};
+
+const ChannelList: React.FC<ChannelListProps> = ({channels}) => {
+  return (
+    <table className="channel-list">
+      <tbody>
+        <tr>
+          <td className={'channel-header'}>Channel ID</td>
+          <td className={'channel-header'}>Leecher</td>
+          <td className={'channel-header'}>Balance</td>
+          <td className={'channel-header'}>Seeder</td>
+          <td className={'channel-header'}>Balance</td>
```

Comment:
We should only be concerned with (and get to know about) channels that we participate in, and those channels are going to be uni-directional, 2-party payment channels. This implies that I am either the seeder or the leecher in any given channel that will be displayed.  So it might be more clearer to have two tables: i) "Receiving channels" and ii) "Outlaying channels" (or similar names). i) would have a 'leecher' column only, and ii) would have a 'seeder' column only.

Reply:
Of course the other channels of relevance to this app are the ledger channels that connect each peer with a payment hub. We probably want to display information about those, too.

Reply:
Yeah, I refactored the component as suggested and show two tables instead: Receiving and Outlaying channels. See the updated screenshot.